src/main/java/net/glowstone/block/blocktype/BlockLiquid.java

```diff
@@ -246,6 +247,9 @@ public void updatePhysics(GlowBlock block) {
         if (!(Byte.compare(block.getState().getRawData(), isWater(block.getType()) || block.getBiome() == Biome.HELL ? STRENGTH_MIN_WATER : STRENGTH_MIN_LAVA) == 0) || block.getRelative(DOWN).getType() == Material.AIR) {
             calculateFlow(block);
         }
+        if (initalData != block.getState().getRawData()) {
```

Comment:
That is supposed to check if there is a change to a liquid block and if so then it will tick again (after waiting the proper amount of time). Does it look like I didn't implement it correctly? If the data didn't change then the block clearly doesn't require ticking again until a block changes around it.